### ISSUE TITLE
Timeline header at the bottom

### ISSUE THREAD

**Issue by MariiaPolova** [namespace-ee/react-calendar-timeline#542]
Hi there!
I didn't find any similar and direct question to that: can top timeline header be placed to the bottom?

**Comment by dkarnutsch** [comment]
I don't think that is possible using the API. However you could try to create a second timeline. Link them together using [Example Linked Timelines](https://github.com/namespace-ee/react-calendar-timeline/tree/master/examples#linked-timelines) and hide the header of the first one (which will be easier with the custom headers coming soon). 

No guarantee though that this works and it seems also rather hacky.

**Comment by tmbrlkV** [comment]
This works for me:
```
.react-calendar-timeline {
    display: flex;
    flex-direction: column;
}

.rtc-outer {
    order: 1;
}

.rct-header-root {
    order: 2;
}
```

**Comment by developer-ocansey** [comment]
Closing as part of issue triage. This issue was created before 2023 and has had no recent activity. If you're still experiencing this on v0.30+, please open a new issue with a minimal reproduction.
